ingestion/src/metadata/test_suite/api/workflow.py

```diff
@@ -219,30 +222,57 @@ def _get_profile_query(self, entity: Table) -> Optional[str]:
 
         return None
 
-    def _get_partition_details(self, entity: Table) -> Optional[TablePartitionConfig]:
+    def _get_partition_details(
```

Comment:
should we have a PartitionsMixin to centralise the partitioning details? I see we are repeating the logic for profiler and Test Suite

Reply:
I am actually creating a util function. It does not even use `self`

Reply:
oh that's right, then we can directly move it out